about.html

```diff
@@ -0,0 +1,197 @@
+                            <img class="img-fluid text-center img-separator" src="./images/2023.jpg" alt="picture">
+                        </div>
+                        <div class="col col-12 col-md-6">
+                            <img class="img-fluid text-center img-separator" src="./images/2022.jpg" alt="picture">
+                        </div>
+                    </div>
+                </div>               
+            </div>
+            
+        </div>
+    </section>
+
+    <section id="partner" class="partner container-fluid">
+        <div class="container">
+            <div class="row justify-content-center">
+                <h2 class="program-title white">Partners</h2>
+                <hr class="hr-program"/>
+            </div>
+            <div class="row justify-content-center align-items-center">
+                <div class="col col-4 col-md-2 text-center ">
+                    <img class="img-fluid" src="./images/p1.jpg" alt="logo-partner">
+                </div>
+                <div class="col col-4 col-md-2 text-center">
+                    <img class="img-fluid" src="./images/p2.jpg" alt="logo-partner">
+                </div>
+                <div class="col col-4 col-md-2 text-center">
+                    <img class="img-fluid" src="./images/p3.jpg" alt="logo-partner">
+                </div>
+                <div class="col col-4 col-md-2 text-center">
+                    <img class="img-fluid" src="./images/p4.jpg" alt="logo-partner">
+                </div>
+                <div class="col col-4 col-md-2 text-center">
+                    <img class="img-fluid" src="./images/p5.jpg" alt="logo-partner">
+                </div>
+                <div class="col col-4 col-md-2 text-center">
+                    <img class="img-fluid" src="./images/p6.jpg" alt="logo-partner">
+                </div>
+                <div class="col col-4 col-md-2 text-center">
+                    <img class="img-fluid" src="./images/p7.jpg" alt="logo-partner">
+                </div>
+                <div class="col col-4 col-md-2 text-center">
+                    <img class="img-fluid" src="./images/p8.jpg" alt="logo-partner">
+                </div>
+            </div>            
+        </div>      
+    </section>
+
+    <footer id="footer" class="container-fluid">
+        <div class="container justify-content-center align-items-center">
+            <div class="row">
+                <div class="col col-6 col-md-4 align-items-center text-center">
+                    <img class="img-fluid logo" src="./images/logo1.jpg" alt="logo">
+                </div>
+                <div class="col col-6 col-md-8 text-left">
+                    <p class="footer-text align-items-center">2023 International Business School. Some Rights Reserved. <br> DISCLAIMER: This page is created for educational purposes, and none of the information here is real.</p>
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
Nice work on the footers😉 But let's match the given design as required for this milestone😉 Just as shown in the design, the footer for the[ About page](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) should have a dark color👍  Yours is currently having a white color like the homepage🥲

<img width="563" alt="image" src="https://user-images.githubusercontent.com/64643995/233151635-8461f403-4c65-409b-afce-9ae82142be97.png">